jest.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
All of the things related to tests can be removed since that's not really what users are here for

Reply:
> 

It was working for me. I'll take a look into it.